en/developer/frameworks/articles/cache-configuration/01-cache-configuration-intro.markdown

```diff
@@ -0,0 +1,155 @@
+---
+header-id: cache-configuration
+---
+
+# Cache Configuration
+
+[TOC levels=1-4]
+
+Caching makes specified objects and object references readily available in
+memory. It costs memory but improves performance. You can experiment with cache
+to determine what's good for your system. If your site serves lots of Web
+Content articles, for example, you may want to cache them.   
+
+There are two cache levels.
```

Comment:
Hi @jhinkey 

I wonder if we should start with "level 1" and "level 2" caches of Hibernate when introducing the overall cache configuration. This may confuse people when they see our Ehcache based caching infrastructure serving as "level X" caches in our parts. For example, Liferay's Freemarker template engine integration uses caching infrastructure as "level 1" and thread locals as "level 2".

Could it be better if we start with general concepts of multi VM pool, single VM pool and configuration files, and then introduce concepts about specific areas?

Reply:
Hi @dantewang ,

I see your point. To simplify things I removed the cache level text and mention that Liferay's data access and template engine components use the configuration framework. I hope that works.

Jim

Reply:
Hi @dantewang - Any more feedback from your review of the two articles in this PR?

- [Cache Configuration intro](https://github.com/jhinkey/liferay-docs/blob/72-cache-config-service-entities/en/developer/frameworks/articles/cache-configuration/01-cache-configuration-intro.markdown)
- [Caching Service Entities](https://github.com/jhinkey/liferay-docs/blob/72-cache-config-service-entities/en/developer/frameworks/articles/cache-configuration/02-caching-service-entities.markdown)